### PR TITLE
Check opencsp.app for doc strings

### PR DESCRIPTION
**Pull request by e10harvey**
Reviewers: Please add suggestions for the 'Placeholder' doc strings or @ mention someone who can fill this in. Thank you.

### CODE REVIEW

File: opencsp/app/camera_calibration/lib/ViewAnnotatedImages.py [hunk 1]
```diff
@@ -101,6 +105,9 @@ def update_image(self):
         self.canvas.itemconfig(self.canvas_image, image=image_tk)
 
     def show_next(self):
+        """
+        Placeholder
```
Comment (e10harvey) [inline opencsp/app/camera_calibration/lib/ViewAnnotatedImages.py:109]:
```suggestion
Show the next image.
```

File: opencsp/app/camera_calibration/lib/ViewAnnotatedImages.py [hunk 2]
```diff
@@ -110,6 +117,9 @@ def show_next(self):
         self.update_image()
 
     def show_prev(self):
+        """
+        Placeholder
```
Comment (e10harvey) [inline opencsp/app/camera_calibration/lib/ViewAnnotatedImages.py:121]:
```suggestion
Show the previous image.
```

File: opencsp/app/camera_calibration/lib/ViewAnnotatedImages.py [hunk 3]
```diff
@@ -77,6 +78,9 @@ def __init__(self, root: tkinter.Tk, images: list[ndarray], image_names: list[st
         self.root.mainloop()
 
     def update_image(self):
+        """
+        Placeholder
```
Comment (braden6521) [inline opencsp/app/camera_calibration/lib/ViewAnnotatedImages.py:82]:
```suggestion
        Updates displayed image and image label
```

File: opencsp/app/sofast/SofastGUI.py [hunk 4]
```diff
@@ -802,6 +802,9 @@ def _projector_closed(self, image_projection: ImageProjection):
         self._camera_closed(None)
 
     def close_image_acquisition(self) -> None:
+        """
+        Placeholder
```
Comment (braden6521) [inline opencsp/app/sofast/SofastGUI.py:806]:
```suggestion
        Close the single Image Acquisition object if open.
```

File: opencsp/app/sofast/lib/ImageCalibrationAbstract.py [hunk 5]
```diff
@@ -49,6 +49,7 @@ def get_calibration_name() -> str:
 
     @abstractmethod
     def apply_to_images(self, measurement) -> ndarray:
+        """Placeholder"""
```
Comment (braden6521) [inline opencsp/app/sofast/lib/ImageCalibrationAbstract.py:52]:
```suggestion
        """
        Performs camera-projector brightness values calibration.

        Parameters
        ----------
        fringe_images : ndarray
            Measurement fringe images.

        Returns
        -------
        ndarray
            Calibrated fringe images, float.
"""
```

File: opencsp/app/target/target_color/target_color_bullseye.py
Comment (braden6521) [inline opencsp/app/target/target_color/target_color_bullseye.py:1]: I do not know what this file does; @rcbrost contributed this. At the moment, the script is broken and does not run. I would recommend removing it from the develop branch if we cannot fix it soon.

File: opencsp/app/target/target_color/target_color_bullseye_error.py [hunk 6]
```diff
@@ -9,27 +9,35 @@
 
 
 def meters_given_pixels(d_pixel, dpi):
+    """
+    Placeholder
+    """
```
Comment (braden6521) [inline opencsp/app/target/target_color/target_color_bullseye_error.py:14]:
```suggestion
    """
    Returns distance in meters given a distance in pixels and a dots per inch (DPI) value
    
    Parameters
    ----------
    d_pixel : float
        Distance in pixels
    dpi : float
        Dots per inch of image

    Returns
    -------
    float
        Distance in meters
    """
```

File: opencsp/app/target/target_color/target_color_bullseye_error.py [hunk 7]
```diff
@@ -9,27 +9,35 @@
...
     radius_of_curvature_meter = 2 * focal_length_meter
     reflected_ray_angle = math.atan(r_meter / radius_of_curvature_meter)  # radians
     surface_normal_error = reflected_ray_angle / 2.0  # radians
     return surface_normal_error * 1000.0  # Convert radians to milliradians.
 
 
 def radius_in_mrad_given_row_col(n_rows, row, col, x_max, cx_offset_pix, y_offset_pix, focal_length_meter):
+    """
+    Placeholder
+    """
```
Comment (braden6521) [inline opencsp/app/target/target_color/target_color_bullseye_error.py:40]: @rcbrost contributed this function. 
**Note**: as of now, this function is broken. Should be removed from develop or fixed.

File: opencsp/common/lib/camera/Camera.py [hunk 8]
```diff
@@ -16,23 +16,24 @@
...
+    image_shape_xy : tuple(int)
+        (x, y), image size in pixels.
+    name : str
+        Name of camera/lens combination.
+
+    """
+
     def __init__(
         self, intrinsic_mat: np.ndarray, distortion_coef: np.ndarray, image_shape_xy: tuple[int, int], name: str
     ):
```
Comment (braden6521) [inline opencsp/common/lib/camera/Camera.py:36]:
```suggestion
    ) -> 'Camera':
    """
    Instantiates a new camera object.

    Parameters
    ----------
    intrinsic_mat : np.ndarray
        3x3 numpy array, intrinsic camera matrix
    distortion_coef : np.ndarray
        1d array, distortion coefficients, typically length 4.
    image_shape_xy : tuple(int)
        (x, y), image size in pixels.
    name : str
        Name of camera/lens combination.
    """
```

File: opencsp/common/lib/camera/Camera.py [hunk 9]
```diff
@@ -16,23 +16,24 @@
...
+    Parameters
+    ----------
+    intrinsic_mat : np.ndarray
+    distortion_coef : np.ndarray
+        1d array, distortion coefficients.
+    image_shape_xy : tuple(int)
+        (x, y), image size in pixels.
+    name : str
+        Name of camera/lens combination.
+
```
Comment (braden6521) [inline opencsp/common/lib/camera/Camera.py:31]:
```suggestion
    Calibrated machine vision camera representation. This represents a pinhole model of
    a camera/lens assembly and includes lens distortion through the use of distortion
    coefficients. This model uses the distortion model used by OpenCV; see
    https://docs.opencv.org/4.9.0/d9/d0c/group__calib3d.html for more information.
```

**Comment by e10harvey** [comment]
This needs a rebase

**Review by braden6521 — APPROVED** [review]
More work is needed in the document department, but this looks good to me!

**Review by bbean23 — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**